h2o-py/h2o/model/model_base.py

```diff
@@ -390,6 +390,20 @@ def scoring_history(self):
         print("No score history for this model")
 
 
+    def ntrees_actual(self):
```

Comment:
I am not sure I like this approach, this adds this method to models where it does not make sense. @michalkurka what do you think?

Reply:
Good point @honza, this is however a common approach - this class already does have algo-specific method.

I would keep this for now.

Reply:
If you want to make a jira - I think we can fix this in the future (we can eg. dynamically remove methods from the instances).

Reply:
:wave: